src/classification/naive_bayes.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import GaussianNB
+from sklearn.metrics import confusion_matrix, accuracy_score, precision_score, recall_score, f1_score
+
+class NBClassifier:
+
+    def __init__(self, dataset_path, dataset_size=400000):
```

Comment:
Use a constant for the _400000_ value instead of direct assignment and add a comment why we took that so that we could refer quickly in the future.